.github/workflows/ios-release-to-org-openfoodfacts-scanner.yml

```diff
@@ -105,7 +105,7 @@ jobs:
           VERSION_CODE: ${{ inputs.VERSION_CODE }}
 
       - name: Build app
-        run: cd ./packages/smooth_app && cd ios && pod update Sentry && pod update GoogleMLKit/BarcodeScanning && cd .. && flutter build ios --release --no-codesign -t lib/entrypoints/ios/main_ios.dart
+        run: cd ./packages/smooth_app && cd ios && rm -rf Pods && rm Podfile.lock && flutter pub get && pod install && cd .. && flutter build ios --release --no-codesign -t lib/entrypoints/ios/main_ios.dart
```

Comment:
I'm guessing `flutter pub get` will fail, as it is called in the `/ios` folder

Reply:
> I'm guessing `flutter pub get` will fail, as it is called in the `/ios` folder

No it's OK, as long as `flutter` is in the `PATH` (which is likely).